dissect/target/tools/shell.py

```diff
@@ -175,44 +139,54 @@ def get_names(self) -> list[str]:
 
         return names
 
-    def default(self, line: str) -> Optional[bool]:
+    def completenames(self, text: str, *ignored) -> list[str]:
+        # https://github.com/python/cpython/blob/3.12/Lib/cmd.py#L247
+        dotext = "do_" + text
+        return [a[3:] for a in self.get_names() + self._aliases if a.startswith(dotext)]
+
+    def check_custom_command_execution(self, line: str) -> bool | None:
```

Comment:
```suggestion
    def _handle_command(self, line: str) -> bool | None:
```
Maybe a bit more concise?